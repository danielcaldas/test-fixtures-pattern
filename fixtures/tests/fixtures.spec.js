
    // WARNING: this file is generated automatically
    const backgroundColorReplacer = require('../../replacer');

    describe('backgroundColorReplacer', () => {
  
        it("not replace color if no match", () => {
          try {
            const { target, options } = require(`../not-replace-color-if-no-match.js`);
            expect(backgroundColorReplacer(target, options)).toMatchSnapshot();
          } catch(error) {
            expect(error).toMatchSnapshot();
          }
        });
      

        it("not replace color if no new color specified", () => {
          try {
            const { target, options } = require(`../not-replace-color-if-no-new-color-specified.js`);
            expect(backgroundColorReplacer(target, options)).toMatchSnapshot();
          } catch(error) {
            expect(error).toMatchSnapshot();
          }
        });
      

        it("replace color when match and opacity defined", () => {
          try {
            const { target, options } = require(`../replace-color-when-match-and-opacity-defined.js`);
            expect(backgroundColorReplacer(target, options)).toMatchSnapshot();
          } catch(error) {
            expect(error).toMatchSnapshot();
          }
        });
      

        it("replace color when match", () => {
          try {
            const { target, options } = require(`../replace-color-when-match.js`);
            expect(backgroundColorReplacer(target, options)).toMatchSnapshot();
          } catch(error) {
            expect(error).toMatchSnapshot();
          }
        });
      });