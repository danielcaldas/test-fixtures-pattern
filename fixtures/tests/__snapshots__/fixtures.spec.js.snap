// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`backgroundColorReplacer not replace color if no match 1`] = `"background-color: rgb(254, 0, 0)"`;

exports[`backgroundColorReplacer not replace color if no new color specified 1`] = `"background-color: rgb(253, 0, 0)"`;

exports[`backgroundColorReplacer replace color when match 1`] = `"background-color: rgb(0, 255, 0)"`;

exports[`backgroundColorReplacer replace color when match and opacity defined 1`] = `"background-color: rgba(0, 255, 0, 0.5)"`;
